app/javascript/stylesheets/components/_table.scss

```diff
@@ -0,0 +1,3 @@
+.ant-table-tbody .ant-table-row:nth-child(odd) td {
+  background-color: #f8f9f9;
+}
```

Comment:
Good stuff.

Reply:
It's better to move the hard-coded HEX value here to `_variables.scss` and use it here.